Meetme.MatchingService/Meetme.MatchingService.Domain/Events/NotificationEvent.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MediatR;
+using Meetme.MatchingService.Domain.Events.Enums;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Meetme.MatchingService.Domain.Events;
+
+public class NotificationEvent : INotification
+{
+	[BsonId]
+	[BsonGuidRepresentation(GuidRepresentation.Standard)]
+	public Guid Id { get; set; }
```

Comment:
MongoDB ids are not guids. They have ObjectId type, but in C# u can use just string